Snakefile

```diff
@@ -82,6 +82,7 @@ def get_metrics():
                 metric_grabbed.append(metric)
```

Comment:
We should add a check here that at least one metric is available

Reply:
See  d84a8a2